README.md

```diff
@@ -1,25 +1,46 @@
 
-Recreate the figures from https://www.biorxiv.org/content/10.1101/505032v3 in 4 easy steps!
+The DeepCell Kiosk paper [(Bannon et al., 2020)](<https://www.biorxiv.org/content/10.1101/505032v3>) presents cost and runtime benchmarks for running a generic image segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b). This repository exist to aid DeepCell Kiosk users in recreating these figures.
+ 
+## Figure Creation
+
+There are two steps to figure creation:
+    1) Generating a full battery of benchmark data for the DeepCell Kiosk under a variety of conditions. See the `Generating Benchmarking Data` section below for details
+    2) Creating the figures from this benchmarking data. See the `Figure Creation` section below for details
 
-N.B.: Before following any of these steps, I'd recommend activating a Python `virtualenv`. It won't affect how easy it is to run this project, probably, but it will make your life easier after you've forgotten about this project and moved on to other Python packages.
+## Generating Benchmarking Data
 
-(To view API documentation for this project online, please go [here](https://deepcell-kiosk-figure-generation.readthedocs.io).)
+This repository can produce the benchmarking figures from the paper, but it expects a full battery of benchmarking output files from all run conditions. It will not produce any figures as-is without output from all runs. 
+ 
+To generate a full battery of benchmarking run data (i.e., varying image numbers and GPU numbers), please complete the following benchmarking runs:
+ 
+     - 3 runs with 1 GPU and 10,000 images
+     - 3 runs with 4 GPUs and 10,000 images
+     - 3 runs with 8 GPUs and 10,000 images
+     - 3 runs with 1 GPU and 100,000 images
+     - 3 runs with 4 GPUs and 100,000 images
+     - 3 runs with 8 GPUs and 100,000 images
+     - 3 run with 1 GPU and 1,000,000 images
+     - 3 run with 4 GPUs and 1,000,000 images
+     - 3 run with 8 GPUs and 1,000,000 images
+ 
+To do this, follow the instructions in [Developer docs of the kiosk-console repo](https://deepcell-kiosk.readthedocs.io/en/master/DEVELOPER.html), in addition to the constraints outlined below in the `Benchmarking Data Settings` subsection.
+ 
+### Benchmarking Data Settings
+ 
+When recreating the figures from the DeepCell Kiosk paper, please observe the condiguration guidelines below. If you'd like to see the exact configuration the Van Valen Lab used for benchmarking, feel free to checkout the `benchmarks` branch of the `kiosk-console` repository.
+ 
+    - In the DeepCell Kiosk paper, benchmarking data was presented for clusters with maxima of 1, 4, and 8 GPUs. Choose the appropriate maximum during DeepCell Kiosk configuration for the benchmarking dataset you would like to recreate
```

Comment:
Formatting seems off in this section.

Reply:
I modified the formatting on this section and hte list of run conditions, since it looked like the same thing was happening there, too.